extension/view/popup/index.html

```diff
@@ -10,23 +10,21 @@
     />
   </head>
   <body>
-    <div class="app">
-      <div class="panel panel--from">
-        <label>
-          <select id="lang-from" name="from" class="lang-select"></select>
-        </label>
-        <textarea id="input" name="input" autofocus></textarea>
-        <div id="suggestion"><span id="translatefrom"></span> <span id="translatelanguage"></span></div>
-      </div>
-      <button class="swap" title="swap">↔️</button>
-      <div class="panel panel--to">
-        <label>
-          <select id="lang-to" name="to" class="lang-select"></select>
-        </label>
-        <textarea id="output" name="output" readonly></textarea>
-      </div>
-      <div class="footer" id="status"></div>
-    </div>
+		<div class="panel panel--from">
```

Comment:
Thanks for the PR @veyndan . Can we just keep the indentation here?

Reply:
Done! Is that just to keep the git diff clean?

Reply:
Thanks!